cfg/atmos.cfg

```diff
@@ -1,22 +1,30 @@
 SYMBOLS {
-    __STACKSIZE__: type = weak, value = $0800; # 2k stack
+    __TAPEHDR__:   type = import;
+    __BASHDR__:    type = import;
+    __PROGFLAG__:  type = weak, value = $00; # $00=BASIC, $80=machine code
+    __AUTORUN__:   type = weak, value = $00; # $C7=run, $00=only load
+    __STACKSIZE__: type = weak, value = $0800; # 2K stack
+    __RAMEND__:    type = weak, value = $9800; # graphics RAM not grabbed
+#   __RAMEND__:    type = weak, value = $B400; # graphics RAM grabbed
 }
 MEMORY {
     ZP:      file = "", define = yes, start = $00E2, size = $001A;
-    TAPEHDR: file = %O, type   = ro,  start = $0000, size = $0011;
-    RAM:     file = %O, define = yes, start = $0500, size = $9300 - __STACKSIZE__;
+    TAPEHDR: file = %O, type   = ro,  start = $0000, size = $001F;
+    BASHEAD: file = %O, define = yes, start = $0501, size = $000D;
+    RAM:     file = %O, define = yes, start = __BASHEAD_LAST__, size = __RAMEND__ - __RAM_START__ - __STACKSIZE__;
 }
 SEGMENTS {
+    ZEROPAGE: load = ZP,      type = zp;
     TAPEHDR:  load = TAPEHDR, type = ro;
+    BASHDR:   load = BASHEAD, type = ro,  define = yes, optional = yes;
     STARTUP:  load = RAM,     type = ro;
     LOWCODE:  load = RAM,     type = ro,                optional = yes;
-    INIT:     load = RAM,     type = ro,  define = yes, optional = yes;
     CODE:     load = RAM,     type = ro;
     RODATA:   load = RAM,     type = ro;
+    INIT:     load = RAM,     type = ro,  define = yes, optional = yes;
     DATA:     load = RAM,     type = rw;
-    ZPSAVE:   load = RAM,     type = bss, define = yes;
+    ZPSAVE:   load = RAM,     type = rw,  define = yes;
```

Comment:
In case I'm not missing the point... Until some time ago the zpsave area was just `DATA` (and thus `rw`) on all targets. Then someone pointed out that this added ~26 zeros to all binaries. Therefore the `ZPSAVE` segment was introduced. So I don't consider it a nice solution to add those zeros again to the binary. My suggestion is to add some dummy/trailer/<...> `rw` segment after the data segment and place a single byte into it.